SONAR-netCDF4/docs/tablePosition_sub_group.adoc

```diff
@@ -0,0 +1,62 @@
+|==============================================================================================================================================================================================================
+|Description |Obligation |Comment
+|Group attributes | |
+|:string |O |System description
+|Dimensions | |
+|time | |can be fixed or unlimited length, as appropriate
+|Coordinate variables | |
+|uint64 time(time) | |Time from position sensor
+|:axis = "T" | |
+|:calendar = "gregorian" | |
+|:long_name =  "Timestamps for position data" | |
+|:standard_name = "time" | |
+|:units = "nanoseconds since 1601-01-01 00:00:00Z" | |
+| | |
+|Variables | |
+|double latitude(time) |M |latitude of the platform reference point in WGS-84 reference system
+|double :valid_range = –90.0, 90.0 | |
+|:standard_name = "Platform latitude" | |
+|:units = "degrees_north" | |
+|:long_name = "latitude" | |
+|double:_FillValue = Double.NaN | |
+| | |
+|double longitude(time) |M |latitude of the platform reference point in WGS-84 reference system
+|double :valid_range = –180.0, 180.0 | |
+|:standard_name = "Platform longitude" | |
+|:units = "degrees_east" | |
+|:long_name = "longitude" | |
+|double:_FillValue = Double.NaN | |
+| | |
+|float heading(time) |MA |Heading refers to the direction a platform is pointing. This may or may not be the direction that the platform actually travels, which is known as its course or track. Any difference between course and heading is due to the motion of the underlying medium, the air or water, or other effects like skidding or slipping. Heading is typically based on compass directions, so 0° (or 360°) indicates a direction toward true North, 90° indicates a direction toward true East, 180° is true South, and 270° is true West.  
```

Comment:
Heading is also in the attitude group. How do reading programs decided which heading to use? 

The description field is different to the heading description in attitude. They should be the same (perhaps merge the best parts from both fields?).

Reply:
I would rather remove it from attitude subgroup. There is no way to have heading without a connection to GPS and this is done by the Position subgroup. (To be honest I left heading thinking about the yaw which is used for aircraft)
Another thing crossed my mind. What to do with sensors that are both attitude and position sensor ? Any idea ?

Reply:
This comes down to what is an instrument and what is a sensor... Or that instruments can be made up of several sensors... 

We have two aims that conflict a little:
1. store data from multiple sensors (some of which produce the same or similar measurements)
2. provide clear directions on which measurements to use for routine processing of data

V1.0 of the convention did 2, but not 1. The proposed v1.1 currently does 1 but not 2. Perhaps we can have 'recommended_X' variables in each beam_group that gives the path and name of the recommended/suggested position, speed, heading, pitch, roll, etc, variables?

Then, if an instrument provided attitude and position, reading software just needs to look at the 'recommended_X' variables to know where to find things and it doesn't matter where the data are, or what exact type of instrument collected the data.

Reply:
Agreed with the distinction between sensor and attitude.
Concerning the capacity to read variables, what's interesting softwares is most of the time to have the attitude and position at the time of the ping. Data in sensor are not stored at the time of the ping but usually at a higher frequency. 
What I will suggest is to have other variables stored in the beamGroup containing position and attitude at the time of the ping (interpolation are usually done by the sounders). (Remind that this is constraint by the fact that the time dimension for the ping is in the beamGroup)
Those variables are the ones used by the sounder and there is no longuer any question about where to find, except for very special case where you need to retrieve source data at higher frequency. 
As soon as you validate this pull request I will create a new one to add those variables. (I could have done that in the same, but I makes it harder to review)
For your information, we use this approach for bathymetry data and we suggested this format for ADCP too.